packages/wallets/src/features/cashier/modules/DepositCrypto/components/DepositCryptoInfoNotice/DepositCryptoInfoNotice.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useActiveWalletAccount } from '@deriv/api-v2';
+import { Localize } from '@deriv-com/translations';
+import { SectionMessage, Text } from '@deriv-com/ui';
+
+const DepositCryptoInfoNotice = () => {
+    const { data: activeWallet } = useActiveWalletAccount();
+
+    if (activeWallet?.currency !== 'tUSDT') return null;
```

Comment:
Isn't it better to let the parent control whether to render `DepositCryptoInfoNotice` or not?

Reply:
Updated